spec/spec-helper.rb

```diff
@@ -0,0 +1,14 @@
+# Requires supporting ruby files with custom matchers and macros, etc,
+# in spec/support/ and its subdirectories.
+Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
```

Comment:
I removed a bunch of stuff but figured this was a common enough pattern to leave it for later. What do you think?